jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/menusearchfield/MenuSearchFieldVariant.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2023 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.flowui.component.menusearchfield;
+
+import com.vaadin.flow.component.shared.ThemeVariant;
+
+/**
+ * Set of theme variants applicable for the {@link MenuSearchField} component.
+ */
+public enum MenuSearchFieldVariant implements ThemeVariant {
+
+    LUMO_SMALL("small"),
```

Comment:
Let's remove `LUMO_` and `MATERIAL_` prefixes. We don't follow this bizarre Vaadin naming convention in your theme variants.